manager/integration/tests/test_node.py

```diff
@@ -1002,6 +1002,28 @@ def test_node_controller_sync_disk_state(client):  # NOQA
                                      CONDITION_STATUS_TRUE)
 
 
+@pytest.mark.skip(reason="TODO")
+@pytest.mark.node  # NOQA
+@pytest.mark.mountdisk # NOQA
+def test_node_default_disk_addedback_with_extra_disk_unmounted(client):  # NOQA
+    """
+    [Node] Test adding default disk back with extra disk is unmounted
+    on the node
+
+    1. Create a host disk extra_disk using Longhorn volume and
```

Comment:
Add backquote (`) around the extra_disk so it's treated as a name. Same below for the last step.